### ISSUE TITLE
Redirect to another route is not happening

### ISSUE THREAD

**Issue by pbollaram** [okta/samples-js-react#221]
Hi,

Followed same as mentioned in example but i need to redirect my app to different route on successful sign in.
So as mentioned in documentation i am doing this below
const login = async () => {
    oktaAuth.signInWithRedirect({ originalUri: '/dashboard' });
  };
  
 but my dashboard component is not loaded. I see that the URL on the browser is getting changed but react component is not loaded. Maybe the router is not detecting the change from oktaAuth
 
 I have the following route specified in my app.
 
 <Route exact path="/dashboard" component={Dashboard} />

**Comment by arvindkrishnakumar-okta** [comment]
Thanks for the post!

@oleksandrpravosudko-okta can you help with this?

**Comment by oleksandrpravosudko-okta** [comment]
Hi @pbollaram,

Does `/dashboard`-bound component load during 'regular' navigation? (i.e. navigating by link or browsing directly onto it)
Would you mind providing details of your router configuration?

**Comment by pbollaram** [comment]
yes about /dashboard. I was able to load it if iIrefresh the page with /dashboard URL or when clicked on route

 <SecureRoute path="/dashboard" component={Dashboard} />

 "react-router-dom": "^5.2.0"

**Comment by oleksandrpravosudko-okta** [comment]
Do you happen to have `history@5.x` installed? There seem to be [reports](https://github.com/ReactTraining/history/issues/804#issuecomment-647019869) of router not working properly with v5.

**Comment by pbollaram** [comment]
No, I am not using that.
